hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -100,6 +102,9 @@ public class DeleteBlocksCommandHandler implements CommandHandler {
   private final long tryLockTimeoutMs;
   private final Map<String, SchemaHandler> schemaHandlers;
 
+  @VisibleForTesting
+  private long deleteCmdWorkerInterval;
```

Comment:
`@VisibleForTesting` is unnecessary, since the member is `private`, not visible to others.

Also, no need to store the interval in both classes.  Let's keep the one in `DeleteCmdWorker`.

```suggestion
```